From 4bfbb5cfe9ef7d5e3198160bf4938f901dc481a2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 18 Sep 2024 08:47:02 +0000
Subject: [PATCH] Inherit from Adwaita

When AdwaitaLegacy is the selected icon theme, inheritance from Adwaita is needed to provide the symbolic icons.
---
 index.theme | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/index.theme b/index.theme
index 0c00055..29db9c9 100644
--- a/index.theme
+++ b/index.theme
@@ -2,7 +2,7 @@
 Name=AdwaitaLegacy
 Comment=Legacy Fullcolor Icons
 Example=folder
-Inherits=hicolor
+Inherits=Adwaita,hicolor
 Hidden=true
 
 # KDE Specific Stuff
-- 
GitLab

